foundation_cms/static/scss/blocks/themes/default/video_block.scss

```diff
@@ -0,0 +1,47 @@
+.video-block {
+  &__container {
+    position: relative;
+    width: 100%;
+    padding-top: 56.25%;
```

Comment:
Found this neat trick to make the video 16:9 while being growing and shrinking with screen size! https://css-tricks.com/aspect-ratio-boxes/